locale/pt-br/texts.po

```diff
@@ -0,0 +1,734 @@
+msgid ""
+msgstr ""
+"Content-Type: text/plain; charset=UTF-8\n"
+"Plural-Forms: nplurals=2; plural=(n!=1);\n"
+
+#.  This should never happen!
+#: src/models.js:24
+msgid "Unknown"
+msgstr "Desconhecido"
+
+#: src/models.js:27
+#, javascript-format
+msgid "Received ${ symbol }"
+msgstr "Recebido ${ symbol }"
+
+#: src/models.js:29
+msgid "Sent ${ symbol }"
+msgstr "Enviado ${ symbol }"
+
+#: src/models.js:31
+#, javascript-format
+msgid "You sent ${ symbol } to yourself"
+msgstr "Você enviou ${ symbol } para você mesmo"
+
+#.  See https://momentjs.com/docs/#/displaying/calendar-time/
+#: src/models.js:41
+msgid "[Today •] HH:mm"
+msgstr "[Hoje •] HH:mm"
+
+#.  See https://momentjs.com/docs/#/displaying/calendar-time/
+#: src/models.js:42
+msgid "[Tomorrow •] HH:mm"
+msgstr "[Amanhã •] HH:mm"
+
+#.  See https://momentjs.com/docs/#/displaying/calendar-time/
+#: src/models.js:43
+msgid "dddd [•] HH:mm"
+msgstr "dddd [•] HH:mm"
+
+#.  See https://momentjs.com/docs/#/displaying/calendar-time/
+#: src/models.js:44
+msgid "[Yesterday •] HH:mm"
+msgstr "[Ontem •] HH:mm"
+
+#.  See https://momentjs.com/docs/#/displaying/calendar-time/
+#: src/models.js:45
+msgid "[Last] dddd [•] HH:mm"
+msgstr "[Última] dddd [•] HH:mm"
+
+#.  See https://momentjs.com/docs/#/displaying/calendar-time/
+#: src/models.js:46
+msgid "DD MMM YYYY [•] HH:mm"
+msgstr "DD MMM YYYY [•] HH:mm"
+
+#: src/utils.js:181
+msgid "Invalid address"
+msgstr "Endereço inválido"
+
+#: src/screens/About.js:43
+msgid "ABOUT"
+msgstr "SOBRE"
+
+#: src/screens/About.js:54
+msgid "This app is developed by Hathor Labs and is distributed for free."
+msgstr ""
+"Este aplicativo foi desenvolvido pela Hathor Labs e é distribuído de graça."
```

Comment:
Why the empty string above?

Reply:
In this case, it wasn't needed. Fixed. Thanks.